string world_part_definition
---
bool success